src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -963,6 +940,38 @@ private static void updateVelocity(Entity e, double scale)
             }
             if (e instanceof LivingEntity) ((LivingEntity) e).setHealth(health);
         });
+
+        put("mayfly", (e, v) -> {
+            if (v == null) {
+                throw new InternalExpressionException("'mayfly' requires parameters");
+            }
+
+            boolean mayfly = v.getBoolean();
+
+            if (e instanceof ServerPlayerEntity player) {
+                player.getAbilities().allowFlying = mayfly;
+                if (!mayfly && player.getAbilities().flying) {
+                    player.getAbilities().flying = false;
+                }
+                player.sendAbilitiesUpdate();
+            }
+        });
+
+        put("fly_speed", (e, v) -> {
+            Float flySpeed;
+
+            if (v == null) {
+                flySpeed = 0.05F;
```

Comment:
This doesn't seem right...

The problem I see is that it seems counter-intuitive that modify with no values would reset it to a default value. I'd instead mention the default value in the docs and remove the null branch.

Else the call to reset to default would look something like `modify(entity, 'fly_speed')`, that doesn't really look right to me.

Reply:
Well actually I'm farily certain that caling modify with no value causes an error, so that branch of code would never trigger.

Reply:
Nope, it's totally fine, for things such as `modify(entity, 'kill')`, modify(entity, 'remove'), and/or some like that.

Reply:
ah oop my bad